server/core/repo.go

```diff
@@ -35,6 +35,7 @@ type (
 		Provider      Provider      `json:"-"`
 		EnvVariables  []EnvVariable `json:"-"`
 		Mounts        []*Mount      `json:"mounts"`
+		Platforms     string        `json:"platforms"`
```

Comment:
This should be an array?
```suggestion
		Platforms     []string        `json:"platforms"`
```

Reply:
Yeah! That will make things easier